.github/dependabot.yml

```diff
@@ -9,3 +9,6 @@ updates:
       - dependency-name: '@uniswap/default-token-list'
       - dependency-name: '@uniswap/token-lists'
       - dependency-name: '@uniswap/widgets'
+    reviewers:
+      - 'just-toby'
```

Comment:
I'd recommend just using `Uniswap/web-tokens`, or else making a new `Uniswap/dependabot` team to deal with dependabot PRs, instead of adding yourself to every PR.

Reply:
You can add yourself as the sole member of `dependabot` if you want, but at least then you can toggle it without an add'l PR.

Reply:
good call